profiles/default/conf/realm.json

```diff
@@ -1621,17 +1649,19 @@
   "resetCredentialsFlow" : "reset credentials",
   "clientAuthenticationFlow" : "clients",
   "attributes" : {
+    "_browser_header.xXSSProtection" : "1; mode=block",
     "_browser_header.xFrameOptions" : "SAMEORIGIN",
-    "failureFactor" : "30",
     "quickLoginCheckMilliSeconds" : "1000",
-    "maxDeltaTimeSeconds" : "43200",
     "displayName" : "BPM Grid",
+    "_browser_header.xRobotsTag" : "none",
+    "maxFailureWaitSeconds" : "900",
+    "minimumQuickLoginWaitSeconds" : "60",
+    "failureFactor" : "30",
+    "maxDeltaTimeSeconds" : "43200",
     "_browser_header.xContentTypeOptions" : "nosniff",
     "bruteForceProtected" : "false",
-    "maxFailureWaitSeconds" : "900",
     "_browser_header.contentSecurityPolicy" : "frame-src 'self'",
-    "minimumQuickLoginWaitSeconds" : "60",
     "waitIncrementSeconds" : "60"
   },
-  "keycloakVersion" : "2.5.1.Final"
-}
+  "keycloakVersion" : "3.1.0.Final"
```

Comment:
When you update Keycloak version please do it so in the dockerfiles of Business Central and Process Server too.

https://github.com/edge-of-tomorrow/cloud-bpm-grid/blob/master/images/business-central/src/main/docker/Dockerfile#L13
https://github.com/edge-of-tomorrow/cloud-bpm-grid/blob/master/images/process-server/src/main/docker/Dockerfile#L23